include/onnxruntime/core/common/logging/logging.h

```diff
@@ -75,6 +75,18 @@ struct Category {
   // TODO: What other high level categories are meaningful? Model? Optimizer? Execution?
 };
 
+enum class TLKeyword : uint64_t {
```

Comment:
Its a good observation and I will place a comment here with the full TraceLogging name but prefer the shorter name for the def. 

As far as the use of Reserved in ETW/TraceLogging keywords, that's a fairly common practice for maintainability and extensibility. To put a bit of space between lower and higher bits to make the more visibly seen (in hex). Also the thinking if in the future if LOGS() had a better category that could be used that mapped to parts of the code and used as ETW keywords instead of just cart-blanche enabling everying in LOGS() with the right level. E.g. In ETW it would be not common to do this and almost every statement to have a category that maps to the keyword (like we do for Session)

Reply:
I could understand using the abbreviation more if this was in a Windows specific file, however it's in shared common code and being more explicit about the 'TL' would make it clearer to other developers. 

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Capturing the rationale behind the reserved fields in the code would be helpful to future developers.

Reply:
It is in the `onnxruntime::logging` namespace though so that does add some context.

Maybe convert the comment to xmldoc so the translation from TLKeyword to the full name is available via intellisense.